### PR TITLE
Get full info about clicked element

### LINKED ISSUE #120: Extend collected data about clicked node in html
Currently, we are collecting only identifiers of clicked elements (such as id, classname and html tag) with content, but we need to collect additional attributes of the node such as placeholder and others that are defined in the element

### PR DESCRIPTION
**Pull request by xitronix**
Closes #120

### CODE REVIEW

File: test/ArcxAnalyticsProvider.test.tsx [hunk 1]
```diff
@@ -177,7 +173,10 @@ describe('(int) ArcxAnalyticxProvider', () => {
 
         expect(socketStub.emit).calledOnceWith(
           'submit-event',
-          getClickEventBody('div#id-for-click.test-classname-1.test-classname-2', 'Text to click'),
+          getClickEventBody(
+            'div#id-for-click.test-classname-1.test-classname-2 [data-testid=track-click] [class=test-classname-1 test-classname-2]',
```
Comment (gtupak) [inline test/ArcxAnalyticsProvider.test.tsx:177]: - Tag `id`s can contain a dot in them, so this format makes it confusing on where the tag ID stops. We should add a delimiter between the id and the list of classes, such as `;`.
- Do not add spaces between attributes `[..]`
- There's no need to add the `class` attribute

As a result of these points, this is what we should expect here:
`div#id-for-click;.test-classname-1.test-classname-2[data-testid=track-click]`

**Review by gtupak — CHANGES_REQUESTED** [review]

File: test/ArcxAnalyticsProvider.test.tsx (hunk 1 above)
Comment (gtupak) [reply test/ArcxAnalyticsProvider.test.tsx:177]: To give you an idea, Heap does it this way, as an example:

```
@div;#modal-root;|@div;.kbmAfw;.sc-hTBuwn;|@div;.jLzChf;.sc-jfTVlA;|@div;|@div;|@div;.bOhFXr;.sc-iOeugr;|@input;.nfYIP;.sc-lbVpMG;[placeholder=Amount];[type=text];|
```

So it separates each tag by a `|`, prefixes each tag with `@`, each id with `#` and separates each item by a `;`. We don't have to necessarily copy them. If you can think of a better way to do it we can go for it

Comment (xitronix) [reply test/ArcxAnalyticsProvider.test.tsx:177]: so maybe we can just move id tag to attributes, will it be clearer?
According to dot, I got this approach from browser identifier:
image
I will remove spaces between attributes

**Review by gtupak — APPROVED** [review]
lgtm

**Pull request merged**